AGENTS.md

```
@@ -0,0 +1,14 @@
+# AGENTS.md
+
+## Cursor Cloud specific instructions
+
+This is the **Fixera frontend** — a Next.js 15 / React 19 app (marketplace web UI). It talks to the `fixera-server` backend. Standard scripts are in `package.json`.
+
+### Services & how to run
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)